ui/src/components/Tasks/Widgets/FieldDescriptionTooltip/FieldDescriptionTooltip.jsx

```diff
@@ -0,0 +1,29 @@
+import { BsInfoCircleFill } from 'react-icons/bs';
+
+import TooltipTrigger from '../../../TooltipTrigger';
+
+/**
+ * @typedef
+ * {Object} FieldDescriptionTooltipProps
+ * @property {string} description - The description text to display in the tooltip.
+ */
+
+/**
+ * FieldDescriptionTooltip is a component that displays a tooltip
+ * with a description when hovering over an info icon.
+ * @param {FieldDescriptionTooltipProps} props
+ * @return {JSX.Element} The rendered tooltip component.
```

Comment:
No need to type the return value for React components; it's better/easier to let the type system infer it. See the React documentation, which never uses `JSX.Element`: https://react.dev/learn/typescript#useful-types